### DISCUSSION TITLE
Add Flows and settings into snapshots

### DISCUSSION THREAD

**Discussion by benjibenoit** [directus/directus#14589] [Archive]
Hello,

I'm creating this thread because I have some needs regarding snapshots. I need to use them to move from dev to prod and I was wondering if it was possible to add flows and some (or all) directus configurations (such as image size) directly into the snapshots.

Thanks

**Comment by rijkvanzanten** [comment]
Flows / project settings aren't currently part of snapshots. Only the database schema (eg collections/fields/relations) are tracked in snapshots/auto migrations.

**Reply by benjibenoit** [reply]
is it possible to think of a system that would allow this to happen?

**Reply by rijkvanzanten** [reply]
Flow configuration is stored in the database, you can technically already use regular export/import to/from CSV or JSON to achieve the same goal 🙂

**Reply by benjibenoit** [reply]
Indeed, I will do so. I think it would be a good feature to let people choose what they export with the snapshots. It could be useful 😊
